packages/main/src/TabContainer.js

```diff
@@ -464,23 +484,50 @@ class TabContainer extends UI5Element {
 		}
 	}
 
-	_onOverflowListItemClick(event) {
+	async _onOverflowListItemClick(event) {
 		event.preventDefault(); // cancel the item selection
 		const { item } = event.detail;
 
+		this.selectedTab = item.associatedTab;
+
+		walk(this.items, tab => {
+			tab.selectedTab = this.selectedTab;
+		});
+
 		this._onItemSelect(item);
-		this.responsivePopover.close();
+		await this.responsivePopover.close();
+		if (this.responsivePopoverSubItems) {
+			await this.responsivePopoverSubItems.close();
+		}
 		this._setItemsForStrip();
-		this.shadowRoot.querySelector(`#${item.id}`).focus();
+
+		const selectedTopLevel = this._getParentTab(this._selectedTab);
+
+		selectedTopLevel.focus();
+	}
+
+	_getAllSubItems(items, result, level) {
+		items.forEach(item => {
+			if (item.hasAttribute("ui5-tab") || item.hasAttribute("ui5-tab-separator")) {
+				item._style = {
+					"padding-left": `${level / 2}rem`,
+				};
```

Comment:
Is it possible to add this during onBeforeRendering, where all other tabs get their private properties set?